src/common/os/posix/divorce.cpp

```diff
@@ -113,6 +113,12 @@ void divorce_terminal(int mask)
 		}
 	}
 
+	if (int dev_null = open("/dev/null", O_RDWR) >= 0)
+	{
+		dup2(dev_null, fileno(stdout));
+		dup2(dev_null, fileno(stderr));
+	}
```

Comment:
What if dev_null is 1 or 2? What will do dup2(1, 1)? 
If dev_null is not 1 or 2 - should it be closed after dup2 calls?
May be let users put stdout/stderr to some not-null file - for example as a tool to debug UDR?

Reply:
>What if dev_null is 1 or 2? What will do dup2(1, 1)?
>If dev_null is not 1 or 2 - should it be closed after dup2 calls?

Will fix it.

>May be let users put stdout/stderr to some not-null file - for example as a tool to debug UDR?

Originally this commit intended to fix *nix only problem caused by guardian. If we want to use output redirecting as some debugging tool it should be more unified and work on all platforms and without guardian. May be then server binary should have ability to redirect its output to /dev/null or any other file from config?

Reply:
@AlexPeshkoff I have updated branch. Please review. I have added config option to allow user define path. Also both classic and super now will behave same way when handling stdout/stderr. Changes are only for posix build for now.

Reply:
For posix everything appears good.